ra2ce/runners/direct_analysis_runner.py

```diff
@@ -58,7 +58,7 @@ def run(
     ) -> list[AnalysisResultWrapper]:
         _analysis_collection = AnalysisCollection.from_config(analysis_config)
         _results = []
-        for analysis in _analysis_collection.direct_analyses:
+        for analysis in _analysis_collection.damages_analyses:
```

Comment:
This can't be merged as long as Sahand's PR is still open (will create conflicts 100% guaranteed)